content/api/plugins/configuration-api.md

```diff
@@ -129,6 +129,14 @@ in the **Settings** tab of the Test Runner.
 
 ### Switch between multiple configuration files
 
+<Alert type="warning">
```

Comment:
Let's add the `::include{file=partials/warning-plugins-file.md}` warning here instead. It'll be usable once https://github.com/cypress-io/cypress-documentation/pull/4129 gets merged

Reply:
@jaffrepaul you should be good to make this change now!